root_consignment_export.tf

```diff
@@ -45,11 +45,14 @@ module "consignment_export_task_policy" {
   name   = "TDRConsignmentExportECSTaskPolicy${title(local.environment)}"
   policy_string = templatefile(
     "./tdr-terraform-modules/iam_policy/templates/consignment_export_task_policy.json.tpl", {
-      environment               = local.environment,
-      titleEnvironment          = title(local.environment),
-      aws_region                = local.region,
-      account                   = data.aws_caller_identity.current.account_id,
-      kms_export_bucket_key_arn = module.s3_external_kms_key.kms_key_arn
+      environment      = local.environment,
+      titleEnvironment = title(local.environment),
+      aws_region       = local.region,
+      account          = data.aws_caller_identity.current.account_id,
+      kms_export_bucket_key_arns = jsonencode([
```

Comment:
Maybe rename `kms_export_bucket_key_arns` to reflect that its not just the export bucket keys any more: `kms_bucket_key_arns`

This will mean changing the corresponding code in the tdr-terraform-modules as well to have consistent naming.